randovania/games/prime1/gui/preset_settings/prime_enemy_stat_randomizer.py

```diff
@@ -0,0 +1,147 @@
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_range_scale_high=self.range_scale_high.value())
+            self._set_up_config(config)
+
+    def _on_spin_changed_range_health_low(self):
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_range_health_low=self.range_health_low.value())
+            self._set_up_config(config)
+
+    def _on_spin_changed_range_health_high(self):
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_range_health_high=self.range_health_high.value())
+            self._set_up_config(config)
+
+    def _on_spin_changed_range_speed_low(self):
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_range_speed_low=self.range_speed_low.value())
+            self._set_up_config(config)
+
+    def _on_spin_changed_range_speed_high(self):
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_range_speed_high=self.range_speed_high.value())
+            self._set_up_config(config)
+
+    def _on_spin_changed_range_damage_low(self):
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_range_damage_low=self.range_damage_low.value())
+            self._set_up_config(config)
+
+    def _on_spin_changed_range_damage_high(self):
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_range_damage_high=self.range_damage_high.value())
+            self._set_up_config(config)
+
+    def _on_spin_changed_range_knockback_low(self):
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_range_knockback_low=self.range_knockback_low.value())
+            self._set_up_config(config)
+
+    def _on_spin_changed_range_knockback_high(self):
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_range_knockback_high=self.range_knockback_high.value())
+            self._set_up_config(config)
+
+    def _on_check_change(self):
+        with self._editor as editor:
+            config = editor.configuration.enemy_attributes
+            config = dataclasses.replace(config, enemy_rando_diff_xyz=self.diff_xyz.isChecked())
+            self._set_up_config(config)
+
+    def _set_up_config(self, config):
```

Comment:
This name is not that great, maybe something like `_set_enemy_attributes_in_editor`?